creditcoin-js/src/examples/loan-cycle.ts

```diff
@@ -8,12 +9,25 @@ import { ethConnection } from './ethereum';
 import { LoanTerms, TransferKind } from '../model';
 import { setupAuthority } from './setup-authority';
 
-export const fullLoanCycleExample = async () => {
+export type RegisteredWallet = {
```

Comment:
As it is custom to nitpick on naming, the name here does not make justice to its content.

Reply:
@CAGS295 which name do you suggest though ?

Reply:
PostAddressRegistrationBelongings. ^^

Reply:
I agree it doesn't capture it, maybe `PostAddressRegistrationInfo`?